src/javascripts/Runners/assetHandlers/scriptFragmentHandler.js

```diff
@@ -1,12 +1,15 @@
 ngapp.run(function(mergeAssetService, assetHelpers, pexService, progressLogger, gameService) {
     let {getOldPath, getNewPath, findGameAssets} = assetHelpers,
-        {forEachPlugin} = mergeAssetService;
+        {forEachPlugin} = mergeAssetService,
+        {log} = progressLogger;
 
     const fragmentGroups = ['QUST', 'INFO', 'SCEN', 'PERK', 'PACK'],
-          fragmentPath = 'VMAD\\Script Fragments\\fileName';
+          fragmentPath = 'VMAD';
```

Comment:
this change doesn't make sense to me.  with this change you're going to have a `fragments` array which contains entries for every record with a VMAD element.

Reply:
This is to allow me to access both the script fragments, and the scripts from the same handler later on - do you know of a better way?

Can I use `..` in the path?

Reply:
you could store a handle for the record in addition to a handle for the script fragment fileName element.

Reply:
Sounds good - I assume there's a way to get the record handle from a subrecord handle?